cmd/oceantv/broadcast_machine.go

```diff
@@ -170,6 +172,26 @@ func (sm *broadcastStateMachine) handleGoodHealthEvent(event goodHealthEvent) er
 	return nil
 }
 
+var errCtrlNotReport = errors.New("Controller not reporting. Likely caused by low voltage.")
```

Comment:
Conventional in go to have lower case errors. We can probably just shorten to "controller not reporting" as well. It's kind of implicit for us what the cause is i.e. low voltage or just broken hehe

Reply:
Besides, we don't really know the cause of why the controller not reporting. It is best to keep speculation out of error messages and report only what we know.